package.json

```diff
@@ -15,5 +15,8 @@
     "mocha-junit-reporter": "^1.23.3",
     "mochawesome": "^7.1.3",
     "typescript": "^5.0.2"
+  },
+  "devDependencies": {
```

Comment:
Please move the lib to dependencies rather than devDependencies. 

https://www.geeksforgeeks.org/difference-between-dependencies-devdependencies-and-peerdependencies/